### PR TITLE
Gui

### PR DESCRIPTION
**Pull request by YaffaLevy**
game

### CODE REVIEW

File: src/main/java/levy/gameoflife/GameOfLifeComponent.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+
+        for (int i = 0; i < game.getHeight(); i++) {
+            for (int j = 0; j < game.getWidth(); j++) {
+                if (game.getCellState(i, j) == 1) {
+                    g.setColor(Color.BLACK);
+                } else {
+                    g.setColor(Color.WHITE);
```
Comment (andrewoid) [inline src/main/java/levy/gameoflife/GameOfLifeComponent.java]: Paint the whole window white, then you only need to draw the white fill rects

File: src/main/java/levy/gameoflife/Main.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package levy.gameoflife;
+
+public class Main {   public static void main(String[] args) { new GameOfLifeFrame().setVisible(true);
+}
+}
```
Comment (andrewoid) [inline src/main/java/levy/gameoflife/Main.java]: Bad formatting.

Feels like you copied it from somewhere., you obviously didn't write it like this.

File: src/main/java/levy/gameoflife/GameOfLifeFrame.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        initialGrid[20][24] = 1;
+        initialGrid[20][25] = 1;
+        initialGrid[20][26] = 1;
+        game.setGrid(initialGrid);
+
+        setLayout(new BorderLayout());
+
+        GameOfLifeComponent gameComponent = new GameOfLifeComponent(game);
+        add(gameComponent, BorderLayout.CENTER);
+    }
```
Comment (andrewoid) [inline src/main/java/levy/gameoflife/GameOfLifeFrame.java:52]: I asked for a Play and Pause button as well.

File: src/main/java/levy/gameoflife/GameOfLifeFrame.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+public class GameOfLifeFrame extends JFrame {
+
+    private final GameOfLife game = new GameOfLife(50, 40);
+
+    public GameOfLifeFrame() {
+        setSize(800, 600);
+        setTitle("Game of Life");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        int[][] initialGrid = new int[40][50];
```
Comment (andrewoid) [inline src/main/java/levy/gameoflife/GameOfLifeFrame.java]: You are already creating the 50x40 grid in GameOfLife, why are you creating it again?

File: src/main/java/levy/gameoflife/Main.java (hunk 2 above)
Comment (YaffaLevy) [reply src/main/java/levy/gameoflife/Main.java]: I copied from my sand project

Comment (YaffaLevy) [reply src/main/java/levy/gameoflife/Main.java]: thats how i have it in there but i will fix

File: src/main/java/levy/gameoflife/GameOfLifeFrame.java (hunk 3 above)
Comment (YaffaLevy) [reply src/main/java/levy/gameoflife/GameOfLifeFrame.java:52]: forgot sorry- how do you create the symbol? found something called unicode is that what you suggest?

Comment (andrewoid) [reply src/main/java/levy/gameoflife/GameOfLifeFrame.java:52]: However you find to do it.

Comment (YaffaLevy) [reply src/main/java/levy/gameoflife/GameOfLifeFrame.java:52]: added

**Review by andrewoid — APPROVED** [review]

**Pull request merged**